app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+from pathlib import Path
+
+
+def move_file(command: str) -> None:
+    splitted_all = command.split("/")
+    splitted_first_str = str(splitted_all[0]).split()
```

Comment:
```suggestion
    command, source_path, destination_path = splitted_all[0].split()
```

Reply:
splitting into 3 elements won't work correctly with any string, e.g. if the string has only one filename or only command keyword or two filenames without a command, etc., that's why I use a different approach for dissimilar situations.

Reply:
the command will work correctly only if it consists of three points, all other cases will be false, so that it does not cause an error, you can use `try ... except`

Reply:
please give an example of when this will not work